scripts/south_africa/covid19_jhb/readme.md

```diff
@@ -0,0 +1,65 @@
+# Importing COVID19 Data for Johannesburg Into Data Commons
+
+Author: Data Science for Social Impact
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [About the Import](#about-the-import)
+
+## About the Dataset
+
+
+### Download URL
+
+CSV file is available for download from https://github.com/dsfsi/covid19za/blob/master/data/district_data/gp_johannesburg.csv.
+The dataset is called `gp_johannesburg.csv`.
+
+### Overview
+
+This dataset is part of a larger dataset (Coronavirus COVID-19 (2019-nCoV) Data Repository for South Africa), which is created, maintained and hosted by Data Science for Social Impact research group. This dataset is a subset containing COVID-19 data for the City of Johannesburg.
+
```

Comment:
Two questions:

* I notice that the data ends in ~2022, is there any later data?

* any chance of adding more of the places too?

Reply:
- So there is not any later data at the moment.
- We used this import as an initial test to understand how the D/C import process works, thus we are busy working with other more important data that we are working on importing. So we will not be adding any other places in the near future at least